### DISCUSSION TITLE
Solargraph

### DISCUSSION THREAD

**Discussion by boc-tothefuture** [boc-tothefuture/openhab-jruby#142] [General]
@jimtng and @pacive 
I added [solargraph](https://github.com/castwide/solargraph) to the Gemfile.  With a vscode [plugin](https://marketplace.visualstudio.com/items?itemName=castwide.solargraph) this should provide code completion.

**Comment by jimtng** [comment]
I've just stumbled across this and just tried solargraph. Better late than never! Not bad! Thanks for the tip.

I came across rubocop-daemon a few months ago after surviving for so long with a very slow file save.
